charts/zookeeper-operator/values.yaml

```diff
@@ -47,3 +47,18 @@ affinity: {}
 tolerations: []
 # Pod annotations
 annotations: {}
+
+## AdditionalSidecar configuration details.
+additionalSidecars: {}
+# - name: nginx
+#   image: nginx:latest
+
+## AdditionalSidecarEnvironment configuration details.
+additionalSidecarEnv: {}
```

Comment:
These are env's for the zk container, not sidecars.

Reply:
Updated additionalSidecarEnv to additionalEnv